content/projects/java-specific/introduction-to-spring-boot/part-2/_index.md

```diff
@@ -13,124 +13,98 @@ ready: true
 submission_type: continue_repo
 tags:
 - spring-boot
+- rest-api
+- mvc
 - annotations
-- unit-testing
-- caching
-- security
 title: Introduction to Spring Boot - part 2
 ---
 
-We are going to work on Spring boot **Caching** and **Security** for this project
+We are going to focus on creating a REST api that will serve as a end point to our sping boot java application.
```

Comment:
Typo
creating a REST api that will serve as an end point to our spring boot application